siad/daemon.go

```diff
@@ -189,7 +189,7 @@ func startDaemon(config Config) (err error) {
 func startDaemonCmd(cmd *cobra.Command, _ []string) {
 	// Create the profiling directory if profiling is enabled.
 	if globalConfig.Siad.Profile {
-		go profile.StartContinuousProfile(globalConfig.Siad.ProfileDir)
+		go profile.StartContinuousProfile(globalConfig.Siad.ProfileDir, false, false)
```

Comment:
yeah, I don't think there's much value in providing profiling capability in our release binaries. Profiling is generally something you only do after you've identified a problem. (As opposed to logging, which you do before you identify the problem.)